apps/challenges/urls.py

```diff
@@ -33,5 +33,8 @@
         name='get_all_submissions_of_challenge'),
     url(r'(?P<challenge_pk>[0-9]+)/download_all_submissions_file/(?P<file_type>[A-Za-z]+)/$',
         views.download_all_submissions_file,
-        name='download_all_submissions_file')
+        name='download_all_submissions_file'),
+    url(r'challenges/reload_rabbitmq_submission_worker/$',
```

Comment:
Submission worker is a python script. Rabbit mq is a messaging broker. I dont think there is a need to include `rabbitmq` in urls. In fact make the end point as 

```
challenges/submission_worker/reload
```

This way in future(not sure here) if we want to add an option for restart, we could simply expose `/restart` endpoint or a better way can be to take it as keyword argument in the urls like 

```
challenges/submission_worker/:action
```